content/docs/reference/config/installation-options-version-comparison/index.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Installation Options - Generate differences between current release and previous release (v1.0)
+description: Describes the differences between the current and previous releases of installation options from the Helm charts.
+weight: 30
+keywords: [kubernetes,helm]
+---
+
```

Comment:
need some type of introduction here.  The reader will see a table and have no idea what they are looking at :)

Reply:
done